packages/contracts/src/actions/getWalletInstance.ts

```diff
@@ -0,0 +1,13 @@
+import { Wallet } from '@fuel-ts/wallet';
+
+import type { ContractsConfig } from '../types';
+
+export async function getWalletInstance(config: ContractsConfig) {
```

Comment:
We should allow users to deploy with mnemonics as well.  This could be automatically detected by checking if the env `WALLET_SECRET` includes a space.